content/behind_the_scenes/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Behind The Scenes"
+menu:
+  main:
+    weight: 1
+showDate: false
+draft: false
+---
+
+## Overview ##
+
+This article aims to explain how we have re-designed and re-written the backend used to collect, enrich and display the ADSB data our community sends us. 
+
+## Anatomy of a Feeder Connection ##
+When you follow our [docker_planewatch] 
+![plane.watch feeder connection anatomy](Connection%20Anatomy.drawio.png)
+
+1. [pw-feeder][pw-feeder] opens two connections to [Bordercontrol](#bordercontrol) - one connection for [BEAST][beast protocol], and another for [MLAT][mlat-client].
+2. During connection establishment, [Bordercontrol](#bordercontrol) checks the validity of the feeder's API key. If invalid, the connection is dropped. If valid, the connection is allowed, and [bordercontrol](#bordercontrol) starts a "feed-in" container for the client.
+3. Bordercontrol proxies [BEAST][beast protocol] traffic to the feed-in container.
+4. Bordercontrol proxies [MLAT][mlat-client] traffic to an [mlat_server][mlat-server] instance running on the regional multiplexer.
+5. For the "new" environment, [pw_ingest](#pw_ingest) running within the feed-in container decodes the [BEAST][beast protocol] data, and publishes the data as a message onto the NATS message bus. The data is processed through the [pw-pipeline][pw-pipeline].
+6. For the "legacy" environment, the [BEAST][beast protocol] data for each region is multiplexed in each regional multiplexer. Virtual Radar Server then consumes this data for the legacy front-end.
```

Comment:
Add a 7. Data is fetched (via a websocket)by the web interface for displaying to the end users....

You dont actually mention the front end anywhere until the pw-router section.

Reply:
Added an extra point.

Reply:
@mikenye can you update the diagram to remove the legacy stuff and add a point 6 for the frontend part of it?

Reply:
@tardoe done. Updated the list slightly, would suggest re-review. :-)